site/zenodo_rdm/stats/utils.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Statistics utilities."""
+
+import itertools
+
+from invenio_rdm_records.proxies import current_rdm_records_service
+
+try:
+    from functools import lru_cache
+except ImportError:
+    from functools32 import lru_cache
```

Comment:
```suggestion
from functools import lru_cache
```

I think this was a Python 2/3 compatibility trick, so not needed anymore